tests/Functional/Schema/SQLServerSchemaManagerTest.php

```diff
@@ -404,4 +404,11 @@ public function testPkOrdering(): void
         self::assertEquals('colB', $columns[0]);
         self::assertEquals('colA', $columns[1]);
     }
+
+    public function testConcat(): void
+    {
+        $result = $this->connection->fetchOne('SELECT CONCAT(1, 2, 3)');
+
+        self::assertEquals('123', $result);
+    }
```

Comment:
@morozov: I'm not that familiar with the Doctrine test suites, is this an appropriate place for the platform-specific test?

Reply:
It's sort of the right place but it would be better to create a separate test that would run on all supported platforms. Please see the ones located in [tests/Functional/Platform](https://github.com/doctrine/dbal/tree/3.1.x/tests/Functional/Platform) for example.